packages/nimble-components/src/table/template.ts

```diff
@@ -67,7 +67,7 @@ export const template = html<Table>`
                                         ${ref('selectionCheckbox')}
                                         ${'' /* tabindex managed dynamically by KeyboardNavigationManager */}
                                         tabindex="-1"
-                                        class="${x => `selection-checkbox ${x.selectionMode ? x.selectionMode : ''}`}"
+                                        class="selection-checkbox"
```

Comment:
Note: This change isn't part of the "Collapse all" modification. I just noticed that we were adding extra dynamic styling that wasn't necessary, so I removed it.